src/main/java/bio/terra/janitor/service/migirate/MigrateService.java

```diff
@@ -0,0 +1,83 @@
+package bio.terra.janitor.service.migirate;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import javax.sql.DataSource;
+import liquibase.Contexts;
+import liquibase.Liquibase;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.exception.LiquibaseException;
+import liquibase.lockservice.DatabaseChangeLogLock;
+import liquibase.resource.ClassLoaderResourceAccessor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+/**
+ * The MigrateService invokes Liquibase to configure the service database. It provides the methods
+ * to clear stale locks, drop all data, and update the service database. We implement this in code,
+ * as opposed to using hooks in the Spring framework. We expect MC Terra services to scale
+ * horizontally and to perform rolling upgrades of the running instances. Therefore, we do not want
+ * the act of running an instance to suddenly upgrade the database schema.
+ */
+@Component
+public class MigrateService {
```

Comment:
This feels like it should go in some common terra library repo. but I guess let's do that later.